scss/modules/_grid.scss

```diff
@@ -49,6 +49,11 @@ $single-column-percent: $single-column-width / $columns-max-width * 100%;
   @include generate-append-classes($columns, 'append-', '');
   @include generate-push-classes($columns, 'push-', '');
   @include generate-pull-classes($columns, 'pull-', '');
+
+  // The base grid class without a width for adding vertical rhythm.
+  .col {
+    @extend %vf-column;
+  }
```

Comment:
This has exactly the same css as .twelve-col.
1) Do we need both?
2) Shouldn't the rules be merged?